commands/initial/templates/tasks/test.js

```diff
@@ -62,5 +57,7 @@ const getAllConfig = (watch) => ({
 module.exports = run;
 
 if (!module.parent) {
-    run(process.argv[2]);
+	//skip the first two args (exe and script) and grab all options that start with a 'word'
+	let optionArgs = process.argv.filter((value, index) => index > 1 && value.match(/^\w+/));	
```

Comment:
You might be able to combine this with `options.js`, not completely sure though

Reply:
I'm not sure if you saw my comments about `options.js` but since the original code did not use it, I would like to update that in a separate PR. The reason for this is because as I was implementing, the code got messier and I'm sure this wasn't the intention for using it. I'm probably looking at it from the wrong perspective and would like to  understand the intent behind its design but I also feel that it shouldn't hold this PR up while we discuss. Do you agree with this?